src/Aspire.ProjectTemplates/templates/aspire-apphost/.template.config/localize/templatestrings.cs.json

```diff
@@ -13,6 +13,7 @@
   "symbols/skipRestore/description": "Pokud se tato možnost zadá, přeskočí automatické obnovení projektu při vytvoření.",
   "symbols/NoHttps/description": "Určuje, jestli se má vypnout protokol HTTPS.",
   "postActions/set-startup-project/description": "Nastaví projekt po spuštění v řešení",
+  "postActions/set-startup-project/manualInstructions/default/text": "Set the startup project in the solution",
```

Comment:
Would it be acceptable to just copy the similar string from `description` from which I only changed "Sets" to "Set"?